build/operations/services/create-asp.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```

Comment:
same here, should we avoid the abbreviation and name this file "create-app-service-plan"?

Reply:
Normally I would agree not to use abbreviations. In the context of Azure these are well known https://docs.microsoft.com/en-us/azure/cloud-adoption-framework/ready/azure-best-practices/naming-and-tagging and in addition to my other comment felt like the right choice for now.